great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -290,6 +290,12 @@ def to_json_dict(self):
                 if isinstance(val.get("batch_request"), BatchRequest):
                     val["batch_request"] = val["batch_request"].to_json_dict()
 
+        if hasattr(self.checkpoint_config, "batch_request"):
+            if isinstance(self.checkpoint_config["batch_request"], BatchRequest):
```

Comment:
@NathanFarmer I am just curious as to how the `checkpoint_config["batch_request"]` gets to be of type `BatchRequest` rather than a `dict`...  This may be the part that I am forgetting...  Happy to discuss.  Thank you!

Reply:
@alexsherstinsky See my other comment, but the short version is that it was passed into `Checkpoint` as `BatchRequest` and never changed.

Reply:
@alexsherstinsky See the latest changes which convert `BatchRequest` objects to `dict` at `CheckpointConfig` instantiation.